www/templates/forms.html

```diff
@@ -293,6 +293,11 @@
   {% endif %}
      
      <input class="full-name-js" type='hidden' name='comments' />
+
+     {% if build_chain_top_build_url %}
+        <a class="grey-btn" href="{{ build_chain_top_build_url }}">top build</a>
```

Comment:
I think top build option should be outside of Resubmit Build

Reply:
Yes, I think I added it in wrong place, I'll commit a fix for that.